modules/misunderstood/src/views/full/dates.ts

```diff
@@ -0,0 +1,39 @@
+import moment from 'moment'
+
+export const now = new Date()
+export const thisWeek = moment()
+  .startOf('week')
+  .toDate()
+export const lastWeekStart = moment()
+  .subtract(1, 'weeks')
+  .startOf('week')
+  .toDate()
+export const lastWeekEnd = moment()
+  .subtract(1, 'weeks')
+  .endOf('week')
+  .toDate()
+export const thisMonth = moment()
+  .startOf('month')
+  .toDate()
+export const lastMonthStart = moment()
+  .subtract(1, 'months')
+  .startOf('month')
+  .toDate()
+export const lastMonthEnd = moment()
+  .subtract(1, 'months')
+  .endOf('month')
+  .toDate()
+export const thisYear = moment()
+  .startOf('year')
+  .toDate()
+export const lastYearStart = moment()
+  .subtract(1, 'years')
+  .startOf('year')
+  .toDate()
+export const lastYearEnd = moment()
+  .subtract(1, 'years')
+  .endOf('year')
+  .toDate()
+export const last7days = moment()
+  .subtract(7, 'days')
+  .toDate()
```

Comment:
This looks like a duplicate from analytics, what do you think of moving it to common/dates.ts, and export a single object with those properties, instead of multiple exports ?

Reply:
Will do